openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -0,0 +1,19 @@
+"""
+Waffle flags and switches for user authn.
+"""
+from __future__ import absolute_import
+
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace
+
+WAFFLE_NAMESPACE = u'user_authn'
+
+# Switches
+# If this switch is enabled then `edx.org` users must be sign in using their `edx.org` google account
```

Comment:
This probably shouldn't reference `edx.org` specifically. Also, I tend to feel that waffle isn't a great way of setting up permanent features. If it's a feature that can be configured on or off, I think that should live in django settings. I tend to just use waffle for code that I'm looking to incrementally deploy.

Reply:
We need this switch for thorough testing on stage before enabling it on production.